tests/improved-sanity-test/main.yml

```diff
@@ -58,6 +58,10 @@
       tags:
         - redhat_subscription
 
+    - role: docker_pull_run_remove
+      tags:
+        - docker_pull_run_remove
+
```

Comment:
Since this role removes images right after pulling and running them, I don't know how much value is added by running it twice considering the amount of time it takes to pull from docker.io. WDYT?